source/manage/cloud-reporting.rst

```diff
@@ -76,13 +76,13 @@ If the statistics page is loading endlessly and you get an error message saying
 Can Team Admins review their own team's statistics?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-System Administrators can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#about-system-roles-beta>`__ to enable Team Administrators to see team statistics.
+System Administrators can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/onboard/system-admin-roles.html#additional-system-admin-roles-e20>`__ to enable Team Administrators to see team statistics.
```

Comment:
```suggestion
System Administrators can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/onboard/system-admin-roles.html>`__ to enable Team Administrators to see team statistics.
```

Reply:
I've corrected this URL because we don't include package/tier names in product documentation links in order to future-proof them. In the future, if E20 were to change, we wouldn't need to address the URL change as well.

Reply:
Thanks for the correction @cwarnermm. Committing it right away! Please review and approve.